src/Common/tests/System/Net/Configuration.Certificates.cs

```diff
@@ -18,6 +18,8 @@ public static partial class Certificates
         {
             private const string CertificatePassword = "testcertificate";
             private const string TestDataFolder = "TestData";
+            private static Mutex m = new Mutex(false, "Global\\CoreFXTest.Configuration.Certificates.LoadPfxCertificate");
```

Comment:
We *could*, just like Windows *could*.  But they believe (and I concur) that aside from tests one isn't very likely to need to parallel import PFX blobs. The workarounds are

 
1. Import once and reuse
1. Use your own locking structure
1. Import with EphemeralKeySet / PKCS12_NO_PERSIST_KEY